packages/scratch-vm/src/extensions/scratch3_pose_hand/index.js

```diff
@@ -460,21 +460,21 @@ class Scratch3PoseNetBlocks {
                 HAND_PART: {
                     acceptReporters: true,
                     items: [
-                        {text: 'thumb', value: 'thumb'},
-                        {text: 'index finger', value: 'indexFinger'},
-                        {text: 'middle finger', value: 'middleFinger'},
-                        {text: 'ring finger', value: 'ringFinger'},
-                        {text: 'pinky', value: 'pinky'},
+                        { text: 'thumb', value: 'thumb' },
+                        { text: 'index finger', value: 'indexFinger' },
+                        { text: 'middle finger', value: 'middleFinger' },
+                        { text: 'ring finger', value: 'ringFinger' },
+                        { text: 'pinky', value: 'pinky' },
                         // {text: 'base of palm', value: 'palmBase'},
                     ]
                 },
                 HAND_SUB_PART: {
                     acceptReporters: true,
                     items: [
```

Comment:
Below are the only functional changes (will need to change my vs code settings to try to limit these whitespace changes). Why does this work??